CHANGELOG.md

```diff
@@ -1,6 +1,143 @@
 # Okta Node SDK Changelog
 
-## 5.1.0
+## 6.0.0
+- [#278](https://github.com/okta/okta-sdk-nodejs/pull/278) Adds Org, Group Schema, Domain API, introduces enum modules. Please see list below for detailed information:
```

Comment:
Would it be possible to auto generate openapi related changelog? We probably can take it as a future automation improvement.